src/main/resources/site/parts/kpiCalculator/kpiCalculator.es6

```diff
@@ -35,6 +39,15 @@ function renderPart(req) {
   const page = getContent()
   const language = getLanguage(page)
   const phrases = language.phrases
+  const config = query({
```

Comment:
Er det ikke bedre å peke på kalkulator-config innholdet i part config enn å basere seg på at det bare skal finnes ett innhold av typen kalkulatorConfig og ikke fler? Skal ikke den innholdstypen være generisk og brukes på fremtidige kalkulatorer?

Reply:
Tanken er at vi skal ha kun ett innhold av typen kalkulator-config hvor vi har data for de forskjellige kalkulatorene. Dette er fordi kalkulatorene ikke har samme antall datakilder og vi må definere for hver kalkulator hvilken kilde de har.